### ISSUE TITLE
Error: "cannot load such file -- google/cloud/storage" when using `require` in Travis

### ISSUE THREAD

**Issue by jywarren** [danger/danger#1121]
# Report

## What did you do?

We're running `require "google/cloud/storage"` from within a Dangerfile. It's in our Gemfile as well, and we run this in the Dangerfile (i guess we shouldn't need to do both but we are trying to debug this):

```ruby
system("gem install google-cloud-storage")
require "google/cloud/storage"
```

## What did you expect to happen?

We expected to be able to require and use the gem. 

## What happened instead?

In https://github.com/publiclab/plots2/pull/5320, we're getting an error `cannot load such file -- google/cloud/storage` when it's run. 

Any idea why we wouldn't be able to require this? Thank you!

## Your Environment

* Which CI are you running on? Travis
* Are you running the latest version of Danger? We believe so, it's via GitHub PR integration. 
* What is your Dangerfile? 

https://github.com/publiclab/plots2/blob/screenshot-test/Dangerfile

```ruby
# more before this

# Store screenshots in Google Cloud
# https://github.com/googleapis/google-cloud-ruby/blob/master/AUTHENTICATION.md#environment-variables
system("gem install google-cloud-storage")
require "google/cloud/storage"

storage = Google::Cloud::Storage.new project_id: "public-lab"
bucket  = storage.bucket "plots2-screenshots"
# more after this
```

Thank you so much for Danger! ❤️

**Comment by jywarren** [comment]
Cross referencing https://github.com/googleapis/google-cloud-ruby/issues/3117 in case it's related.

**Comment by orta** [comment]
I'm not sure if you can install a gem and then directly use it in a runtime after it has started - did you try moving the installation outside of the dangerfile?

**Comment by jywarren** [comment]
Thanks for the response! I installed it with the gemfile outside danger;
should danger be able to read it from there?

**Comment by orta** [comment]
Yeah, that's what Gemfiles do 👍

**Comment by jywarren** [comment]
Oh yes, sorry, I mean it's already installed in the gemfile, and still not
getting picked up. I added the inline install when that didn't work. Is
there something about the syntax of the require using / characters or
something that is tripping this up? I wasn't able to require it in pry
either, and oddly when I ran `gem list | grep google` to see if it was
installed, it wasn't, but outside pry/danger, the same command shows it
installed. What gem set is danger actually using?

Thanks so much for your help!!!
